RestaurantMenu.js

```diff
@@ -1,22 +1,20 @@
 import { EditableMenuEntry } from "/menu/EditableMenuEntry.js";
 import { NewEntryForm } from "/menu/NewEntryForm.js";
-import { ProductsAPI } from "./ProductsAPI.js";
-import { OrdersAPI } from "./OrdersAPI.js";
 import { OrderMenuEntry } from "./OrderMenuEntry.js";
 import { ProvideAddressAndOrderForm } from "./ProvideAddressAndOrderForm.js";
 import { Basket } from "./Basket.js";
+import { ordersAPI, productsAPI } from "./API/index.js";
```

Comment:
```suggestion
import { ordersAPI, productsAPI } from "./API";
```
The whole point of naming files `index` is that it is the default file for a particular directory. Sol you don't need to provide the `index` file name.